### DISCUSSION TITLE
Welcome to cryptoxlib-aio Discussions!

### DISCUSSION THREAD

**Discussion by nardew** [nardew/cryptolib-aio#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grahamers** [comment]
Hi,

First off, thanks so much for this API. Its awesome.

I have a blocking issue. 

I have placed a LIMIT order. Its sitting on order book. 

I need to see its status (filled/pending/rejected etc). Could somebody be kind enough to show me a code snippet where I can poll its status? 

kind regards,


Graham Walsh

**Reply by nardew** [reply]
Hi Graham, which exchange is it? Which call from the exchange's API documentation are you trying to invoke?
